digger/builds/__init__.py

```diff
@@ -22,8 +23,12 @@
     # then we can be sure that it is a Cordova project.
     # we also need to check if we want to build an IOS app or Android app for the Cordova.
     # for now, we don't do that and support Android only.
-    'is_valid': lambda path: os.path.exists('%s/config.xml' % path),
+    'is_valid': lambda path: os.path.exists('%s/www/config.xml' % path) or os.path.exists('%s/config.xml' % path),
```

Comment:
In what case the config.xml file exists in `www` folder, but not in root dir? Or, are both fine? just curious.